cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.stderr.golden

```diff
@@ -4,11 +4,12 @@ USAGE:
   scw baremetal server install <server-id ...> [arg=value ...]
 
 ARGS:
-  server-id             Server ID to install
-  os-id                 ID of the OS to install on the server
-  hostname              Hostname of the server
-  ssh-key-ids.{index}   SSH key IDs authorized on the server
-  [zone=fr-par-1]       Zone to target. If none is passed will use default zone from the config (fr-par-2)
+  server-id              Server ID to install
+  os-id                  ID of the OS to install on the server
+  hostname               Hostname of the server
+  [all-ssh-keys=false]   Add all SSH keys on your baremetal instance
+  ssh-key-ids.{index}    SSH key IDs authorized on the server
```

Comment:
This is a good candidate for oneof mechanism as only one of `all-ssh-keys` or `ssh-key-ids` can be provided. We should either wait for the one of feature to exist or create an issue to track this

Reply:
https://github.com/scaleway/scaleway-cli/pull/1019